packages/server/package.json

```diff
@@ -20,9 +20,9 @@
   "main": "dist/index.js",
   "scripts": {
     "test": "echo 'Please run `npm test` from the root of the repo' && exit 1",
-    "build": "npm run build-js && npm run build-flow",
-    "build-js": "babel-node ../../resources/buildJs.js",
-    "build-flow": "babel-node ../../resources/buildFlow.js",
+    "build": "yarn run build-js && yarn run build-flow",
+    "build-js": "node  ../../resources/buildJs.js",
```

Comment:
Since you're making changes anyway... Could you remove the double-space... Sorry 😬

Reply:
yes, coming back with some cleanup. I've actually used this PR as the final draft towards the monorepo, and have cleared up a lot of these issues there